[2025-03-24T12:35:00.925+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T12:35:01.068+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.download_source_data scheduled__2024-08-02T06:00:00+00:00 [queued]>
[2025-03-24T12:35:01.190+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.download_source_data scheduled__2024-08-02T06:00:00+00:00 [queued]>
[2025-03-24T12:35:01.191+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-24T12:35:01.325+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): download_source_data> on 2024-08-02 06:00:00+00:00
[2025-03-24T12:35:01.346+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3127) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-24T12:35:01.354+0000] {standard_task_runner.py:72} INFO - Started process 3159 to run task
[2025-03-24T12:35:01.415+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'download_source_data', 'scheduled__2024-08-02T06:00:00+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmpzqap27qb']
[2025-03-24T12:35:01.417+0000] {standard_task_runner.py:105} INFO - Job 123: Subtask download_source_data
[2025-03-24T12:35:01.638+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.download_source_data scheduled__2024-08-02T06:00:00+00:00 [running]> on host 4791d3610749
[2025-03-24T12:35:03.093+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='download_source_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-02T06:00:00+00:00'
[2025-03-24T12:35:03.105+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T12:35:03.265+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T12:35:03.291+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 2024-08']
[2025-03-24T12:35:03.514+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T12:35:03.520+0000] {subprocess.py:106} INFO - 2024-08
[2025-03-24T12:35:03.535+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T12:35:03.724+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T12:35:03.725+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=download_source_data, run_id=scheduled__2024-08-02T06:00:00+00:00, execution_date=20240802T060000, start_date=20250324T123501, end_date=20250324T123503
[2025-03-24T12:35:03.974+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T12:35:03.999+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-25T11:09:18.896+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-25T11:09:18.939+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.download_source_data scheduled__2024-08-02T06:00:00+00:00 [queued]>
[2025-03-25T11:09:18.954+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.download_source_data scheduled__2024-08-02T06:00:00+00:00 [queued]>
[2025-03-25T11:09:18.955+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-25T11:09:19.169+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): download_source_data> on 2024-08-02 06:00:00+00:00
[2025-03-25T11:09:19.216+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=92) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-25T11:09:19.219+0000] {standard_task_runner.py:72} INFO - Started process 119 to run task
[2025-03-25T11:09:19.260+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'download_source_data', 'scheduled__2024-08-02T06:00:00+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmpcrzgwf52']
[2025-03-25T11:09:19.269+0000] {standard_task_runner.py:105} INFO - Job 167: Subtask download_source_data
[2025-03-25T11:09:19.602+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.download_source_data scheduled__2024-08-02T06:00:00+00:00 [running]> on host 4c8ab9810a8c
[2025-03-25T11:09:20.410+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='download_source_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-02T06:00:00+00:00'
[2025-03-25T11:09:20.421+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-25T11:09:20.571+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-25T11:09:20.573+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 2024-08']
[2025-03-25T11:09:20.758+0000] {subprocess.py:99} INFO - Output:
[2025-03-25T11:09:20.790+0000] {subprocess.py:106} INFO - 2024-08
[2025-03-25T11:09:20.795+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-25T11:09:20.944+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-25T11:09:20.945+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=download_source_data, run_id=scheduled__2024-08-02T06:00:00+00:00, execution_date=20240802T060000, start_date=20250325T110918, end_date=20250325T110920
[2025-03-25T11:09:21.138+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-25T11:09:21.236+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-25T11:09:21.241+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
